### PR TITLE
cheats, fixes, cosmetic changes

### PR DESCRIPTION
**Pull request by majesticCoding**

### CODE REVIEW

File: src/core/Frontend.cpp [hunk 1]
```diff
@@ -5369,6 +5369,7 @@ CMenuManager::ConstructStatLine(int rowIdx)
 	STAT_LINE("FEST_CC", &CStats::CriminalsCaught, false, nil);
 	STAT_LINE("FEST_FE", &CStats::FiresExtinguished, false, nil);
 	STAT_LINE("DAYPLC", &(nTemp = CTimer::GetTimeInMilliseconds() + 100), false, nil);
+	//TODO(MIAMI): move this function to the CStats and add reading of Stat lines tied with "MEDIA" for the "CHASESTAT" cheatcode
```
Comment (erorcun) [inline src/core/Frontend.cpp:5372]: this function is hella TODO so this comment is redundant actually, but not a problem ofc

File: src/render/Hud.cpp [hunk 2]
```diff
@@ -524,6 +526,54 @@ void CHud::Draw()
 			}
 		}
 
+		if (CStats::ShowChaseStatOnScreen != 0) {
+			static int32 nMediaLevelCounter;
```
Comment (erorcun) [inline src/render/Hud.cpp:530]: this should be initialized to 0, before the if block

File: src/render/Hud.cpp [hunk 3]
```diff
@@ -524,6 +526,54 @@ void CHud::Draw()
...
+
+				CRGBA colour;
+				if (CTimer::GetTimeInMilliseconds() & 0x200)
+					colour = CRGBA(204, 0, 185, 180);
+				else
+					colour = CRGBA(178, 0, 162, 180);
+				CFont::SetColor(colour);
+				CFont::PrintString(SCREEN_SCALE_FROM_RIGHT(110.0f), SCREEN_SCALE_Y(113.0f), gUString);
+				
+				static wchar *prevChaseString;
```
Comment (erorcun) [inline src/render/Hud.cpp:563]: this is not a static

File: src/render/Hud.cpp [hunk 4]
```diff
@@ -524,6 +526,54 @@ void CHud::Draw()
 			}
 		}
 
+		if (CStats::ShowChaseStatOnScreen != 0) {
+			static int32 nMediaLevelCounter;
+			float fCurAttentionLevel = CWorld::Players[CWorld::PlayerInFocus].m_fMediaAttention;
+			if (HUD_TEXT_SCALE_X * CStats::HighestChaseValue > fCurAttentionLevel
```
Comment (erorcun) [inline src/render/Hud.cpp:532]: looks like 0.7 used in here is coincidentally same number with HUD_TEXT_SCALE_X, you can use hardcoded 0.7 in here.

File: src/render/Hud.cpp (hunk 2 above)
Comment (majesticCoding) [reply src/render/Hud.cpp]: fixed

File: src/render/Hud.cpp (hunk 3 above)
Comment (majesticCoding) [reply src/render/Hud.cpp]: fixed

**Review by erorcun — APPROVED** [review]

File: src/core/Pad.cpp [hunk 5]
```diff
@@ -208,15 +209,20 @@ void HealthCheat()
 	}
 }
 
-void VehicleCheat(bool something, int model)
+void VehicleCheat(int model)
 {
 	CHud::SetHelpMessage(TheText.Get("CHEAT1"), true);
-	CStreaming::RequestModel(model, 0);
-	CStreaming::LoadAllRequestedModels(something);
+	CStreaming::RequestModel(model, 1);
```
Comment (aap) [inline src/core/Pad.cpp:215]: use the enum here

**Pull request merged**